python/pyspark/sql/column.py

```diff
@@ -244,9 +251,14 @@ def __contains__(self, item):
     _bitwiseOR_doc = """
     Compute bitwise OR of this expression with another expression.
 
-    :param other: a value or :class:`Column` to calculate bitwise or(|) against
-                  this :class:`Column`.
+    Parameters
+    ----------
+    other
+        a value or :class:`Column` to calculate bitwise or(|) against
```

Comment:
Nit: space before parens, and "with" may be better than "against". In these three bitwise ops